operator/config/deploy/karmada-operator.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: karmada-operator
+  namespace: karmada-system
+  labels:
+    karmada-app: karmada-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      karmada-app: karmada-operator
+  template:
+    metadata:
+      labels:
+        karmada-app: karmada-operator
+    spec:
+      containers:
+      - name: karmada-operator
+        image: release.daocloud.io/karmada/karmada-operator:v0.0.1-rc
```

```suggestion
        image: docker.io/karmada/karmada-operator:latest
```
By the way, I can't see any reference to this yaml, does it is used to demonstrate?